enhancements/builds/disable-jenkins-pipeline.md

```diff
@@ -0,0 +1,299 @@
+As long as the Jenkins Pipeline strategy can be run on a fully supported cluster (i.e upgrades are allowed), existing console integrations with Jenkins should remain in place.
+
+#### Telemetry / Operational Readiness
+
+The feature set will take advantage of existing telemetry tooling.
+Clusters that enable the DeprecatedJenkinsPipeline feature set will be automatically reported to Telemeter.
+Fleet operators/SREs will be able to identify which clusters have the feature set enabled.
+
+The bootstrap behavior will ensure that most clusters which rely on the JenkinsPipeline build strategy have the feature set enabled automatically.
+The exception to this rule will be clusters that have a separate feature set enabled prior to upgrade.
+The following query in Data Hub will reveal which clusters fall into this category:
+
+`count by (_id, name) (cluster_feature_set{name!=""}) * on (_id) group_right() openshift:build_by_strategy:sum{strategy="jenkinspipeline"}`
+
+As of this writing, no clusters fall into this category.
+This may change if clusters opt into tech preview features related to OpenShift builds.
+
+#### Documentation
+
+Documentation related to the Jenkins Pipeline build strategy will need to inform admins that the `DeprecatedJenkinsPipeline` feature set needs to be enabled for builds to succeed.
+
+Examples for Jenkins usage in openshift/origin should be removed.
+Tests which utilized this example deployment should instead instantiate one of the Jenkins templates installed via the Samples operator.
+
+### Risks and Mitigations
+
+**Risk**: Jenkins pipeline builds fail on cluster upgrade, requiring manual intervention.
+
+*Mitigation*: openshift-controller-manager-operator will include bootstrap logic to detect if the cluster uses Jenkins Pipeline builds.
+
+**Risk**: Bootstrap logic can overwrite other enabled features.
+
+*Mitigation*: Bootstrap logic is only invoked if the cluster uses the default feature set.
+
+**Risk:** Clusters need tech preview features AND the Jenkins pipeline build strategy enabled.
+
+*Mitigation*: Clusters with tech preview features are automatically marked as `NoUpgrade` clusters.
+If necessary, a cluster that needs tech preview features and the Jenkins pipeline build strategy can use the `CustomNoUpgrade` feature set,
+which gives admins the ability to enable individual feature gates on their own.
+
+**Risk:** Clusters need fully supported extra features AND the Jenkins pipeline build strategy enabled.
+
+*Mitigation*: There is no real mitigation for this other than using the `CustomNoUpgrade` feature set.
+The downside of this approach is that the cluster cannot be upgraded to the next minor release.
+
+## Design Details
+
+### Open Questions [optional]
+
+1. How many clusters use Jenkins pipeline builds and also enable a supported feature gate?
+   None today, but that may change if other build related features require a feature gate (example - tech preview items).
```

Comment:
shouldn't featuresets/featuregates be additive?

my understanding is:
1) if you have no featureset enabled, you get the default features
2) if you enable techpreview you get the default features + tech preview ones
3) if you enable custom, you get exactly what you define in custom (you do not automatically get the default ones)
4) if you enable DeprecatedJenkins you get default+jenkins

I would expect if you enable multiple featuregates (other than Custom), you would get the union of all the features, no?  So it would only be problematic if some featureset(other than custom) needed to block some of the default features, and you wanted to use it w/ another featureset.


cc @deads2k